source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../variables/breakpoint";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($start: $baselinegrid-space-medium, $end: $baselinegrid-space-medium);
+}
+
+.figure__heading {
+  @include block-spacing($end: 0);
+  @include caption-typography();
+}
+
+.figure__caption {
+  @include caption-typography();
+  @include margin(12px, inline);
+}
+
+@include mq($from: medium) {
+
+  .figure__caption {
```

Comment:
Would prefer the `mq` being inside the existing `.figure__caption` selector.

Reply:
If we need to add anything else to the break point then it will need to be outside of the class so it would need to be reverted back to how it is.

Reply:
It would still be easier to have separate media queries in that case (like [`content-header.scss`](https://github.com/libero/pattern-library/blob/master/source/sass/patterns/organisms/content-header.scss)).

Reply:
If a breakpoint adds something completely new, ok to keep separate. If an existing property is just being changed though, better to keep them together.

Reply:
Good points @thewilkybarkid. We already use this idiom in this repo for the [content header](https://github.com/libero/pattern-library/blob/d31da942b6639ceabef6473f04af48f8344366ad/source/sass/patterns/organisms/content-header.scss#L35).

Reply:
I was thinking of another way. Will explain when you are back in the office @davidcmoulton